recipes/quickfix/all/patches/0005-inc.-required-CMake-version-add-conan-basic-setup-an.patch

```diff
@@ -25,8 +25,8 @@ index 48908ead..8789432c 100644
  endif()
```

Comment:
This patch still contains reference to `CONAN_LIBS`, so it has good chance to have broken logic when dependencies are enabled.

Reply:
Moreover patch name should be changed, because `add-conan-basic-setup` is a conan v1 thing.